.gitignore

```diff
@@ -52,3 +52,5 @@ build/
 
 # buildscripts/buildall.sh, .github/workflows/release.yml
 build-*/
+buildscripts/.DS_Store
+.DS_Store
```

Comment:
Doesn't make sense for mac updates to the gitignore to be in this PR, this should be removed

Reply:
sorry these are auto generated files in macos that dont do anything ans are actually invisible

Reply:
In reality our gitignore should ignore all editor & OS files from any developer, it's fine to include these.